keyboards/littlealby/mute/config.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2019 albybarber
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x5342 // "SB"
+#define PRODUCT_ID      0x3225 // "2%"
```

Comment:
Please change these values, they are already in use by the 2%.

Reply:
@fauxpark Do you know how I generate / get these IDs?

Reply:
Ah, I think I have found it - https://github.com/qmk/qmk_firmware/blob/master/docs/config_options.md#hardware-options

Reply:
In general you can set the VID to `0xFEED` and the PID to the uppercased output of `echo -n "Mute" | shasum | cut -c1-4` (eg. in this case it would be `0x0F09`), but if you want to add VIA support at some point, you'll probably be asked to change it according to their requirements, which is to use the values of two ASCII characters, for example "AB" "MU" = `4142:4D55`.